<launch>
	<node name="TeleopControl" pkg="common_teleop" type="teleop_node" output="screen" respawn="true"></node>
</launch>
